onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -687,13 +696,35 @@ void addGlobalMethods(py::module& m, const Environment& env) {
 
 #ifdef USE_OPENVINO
   m.def(
-      "set_openvino_device", [](const std::string& device) { openvino_device = device; },
-      "Set the prefered OpenVINO device(s) to be used. If left unset, all available devices will be used.");
+      "set_openvino_device", [](const std::string& device_type) { openvino_device_type = device_type; },
```

Comment:
Do we need a separate function exposed for every setting? Didn't we have set_providers with key/val settings?

Reply:
Using Session.set_providers() key/val values now.